docs/user/build-customization.rst

```diff
@@ -4,11 +4,11 @@ Build process customization
 Read the Docs has a :doc:`well-defined build process </builds>` that works for many projects.
 We also allow customization of builds in two ways:
 
-Extend or partially override the build process
+Customize our standard build process
```

Comment:
The headings below should match, and I did like how these items linked to the headings previously.

Reply:
We don't have two sections anymore, it's just one now, since the only difference is to omit the sphinx/mkdocs keys.